src/server/pps/server/api_server.go

```diff
@@ -2609,36 +2612,31 @@ func isNotFoundErr(err error) bool {
 	return err != nil && strings.Contains(err.Error(), "not found")
 }
 
-// pipelineStateToStopped defines what pipeline states are "stopped"
-// states, meaning that pipelines in this state should not be managed
-// by pipelineManager
-func pipelineStateToStopped(state pps.PipelineState) bool {
-	switch state {
-	case pps.PipelineState_PIPELINE_STARTING:
-		return false
-	case pps.PipelineState_PIPELINE_RUNNING:
-		return false
-	case pps.PipelineState_PIPELINE_RESTARTING:
-		return false
-	case pps.PipelineState_PIPELINE_PAUSED:
-		return true
-	case pps.PipelineState_PIPELINE_FAILURE:
-		return true
-	case pps.PipelineState_PIPELINE_STANDBY:
-		return false
-	default:
-		panic(fmt.Sprintf("unrecognized pipeline state: %s", state))
+func (a *apiServer) markPipelineRunning(pachClient *client.APIClient, pipelineName string) error {
+	_, err := col.NewSTM(pachClient.Ctx(), a.etcdClient, func(stm col.STM) error {
+		pipelines := a.pipelines.ReadWrite(stm)
+		pipelinePtr := &pps.EtcdPipelineInfo{}
+		if err := pipelines.Get(pipelineName, pipelinePtr); err != nil {
+			return err
+		}
+		pipelinePtr.State = pps.PipelineState_PIPELINE_RUNNING
+		pipelinePtr.Stopped = false
```

Comment:
Having this set Stopped to false seems a bit weird here, updating the state should be something that happens distinct from the Stopped flag, since one is updating state and the other is updating intent.